ports/glutin/window.rs

```diff
@@ -260,12 +260,21 @@ impl Window {
             Event::Resized(width, height) => {
                 self.event_queue.borrow_mut().push(WindowEvent::Resize(Size2D::typed(width, height)));
             }
-            Event::MouseInput(element_state, mouse_button, _) => {
-                // FIXME(#11130, pcwalton): The third field should be used.
+            Event::MouseInput(element_state, mouse_button, pos) => {
                 if mouse_button == MouseButton::Left ||
-                                    mouse_button == MouseButton::Right {
-                        let mouse_pos = self.mouse_pos.get();
-                        self.handle_mouse(mouse_button, element_state, mouse_pos.x, mouse_pos.y);
+                    mouse_button == MouseButton::Right {
+                        match pos {
+                            Some((x, y)) => {
+                                self.mouse_pos.set(Point2D::new(x, y));
+                                self.event_queue.borrow_mut().push(
+                                    WindowEvent::MouseWindowMoveEventClass(Point2D::typed(x as f32, y as f32)));
+                                self.handle_mouse(mouse_button, element_state, x, y);
+                            }
+                            None => {
+                                let mouse_pos = self.mouse_pos.get();
+                                self.handle_mouse(mouse_button, element_state, mouse_pos.x, mouse_pos.y);
+                            }
+                        }
```

Comment:
I believe this is at a wrong level of indentation. The last closing brace doesn't match up with the `if`.